l10n_be_bpost_address_validation/wizards/bpost_address_validation.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+from ..models.bpost_address import BpostAddress
+
+
+class BpostAddressValidationWizard(models.TransientModel):
+    _name = "bpost.address.validation.wizard"
```

Comment:
@kouffsamuel Could you put a description attribute here ?

Odoo generates warnings : 
![image](https://github.com/OCA/l10n-belgium/assets/19529533/df90f530-e14f-4e9e-a53c-072e7f48929d)

Reply:
I fix it. Is it ok now ?